cmd/root.go

```diff
@@ -105,6 +115,7 @@ func newRootCmd() *cobra.Command {
 	cmd.PersistentFlags().BoolVarP(&configure, "configure", "c", false, "configure initial settings")
 	cmd.PersistentFlags().StringVarP(&profile, "profile", "p", "default", "AWS profile")
 	cmd.PersistentFlags().StringVarP(&roleName, "role", "r", "", "AWS IAM role name")
+	cmd.PersistentFlags().BoolVarP(&web, "web", "w", false, "AWS management console in a browser")
```

Comment:
I thought it would be better to give a verb to make the action explicit (in my opinion).

for example:
```suggestion
	cmd.PersistentFlags().BoolVarP(&web, "web", "w", false, "open AWS management console in a browser")
```

Reply:
Thank you. We will adopt your suggestion.

https://github.com/cybozu/assam/pull/258/commits/50d46d8e2830a8a003a3b7864f46cc0a7bb7fcca#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR118